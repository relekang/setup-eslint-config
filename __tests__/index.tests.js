const { createConfig } = require("../");

test("createConfig with node set in current config", async () => {
  const setupConfig = {
    prompts: [],
    createDependencyList: () => ["eslint"],
    packageInfo: { devDependencies: { eslint: "^42" } },
  };

  expect(
    await createConfig(setupConfig, { env: { node: true } })
  ).toMatchSnapshot();
});

test("createConfig without env in current config", async () => {
  const setupConfig = {
    prompts: [],
    createDependencyList: () => ["eslint"],
    packageInfo: { devDependencies: { eslint: "^42" } },
  };

  expect(await createConfig(setupConfig, {})).toMatchSnapshot();
});
