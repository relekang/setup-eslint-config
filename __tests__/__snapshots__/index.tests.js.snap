// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createConfig with node set in current config 1`] = `
Object {
  "babel": false,
  "dependencies": Array [
    "eslint@^42",
  ],
  "flowtype": false,
  "jest": true,
  "node": true,
  "prettier": true,
  "react": false,
  "typescript": false,
  "vue": false,
  "yarn": false,
}
`;

exports[`createConfig without env in current config 1`] = `
Object {
  "babel": false,
  "dependencies": Array [
    "eslint@^42",
  ],
  "flowtype": false,
  "jest": true,
  "node": false,
  "prettier": true,
  "react": false,
  "typescript": false,
  "vue": false,
  "yarn": false,
}
`;
